signac/contrib/filesystems.py

```diff
@@ -31,7 +31,14 @@
 
 
 def _register_fs_class(fs):
-    "Register a file system handler in the module's registry."
+    """Register a file system handler in the module's registry.
```

Comment:
@ac-optimus We should revert this file and add it to the "ignore" list. We aren't supporting this module in signac 2.0 and have deprecated the entire file. Please add exceptions for any classes, methods, or modules that are marked as deprecated so that we don't waste time adding docstrings to that code.

https://github.com/glotzerlab/signac/blob/fd85b52189fc941d3e25c01d4501c44bb1fcbf64/signac/contrib/filesystems.py#L28-L30

Reply:
Is this file suppose to add to some "ignore" list that already exist? if yes where can I find that?
Also for other classes, method, etc that are marked deprecated I have a doubt. Shouldn't I add a warning as [this](https://numpydoc.readthedocs.io/en/latest/format.html#sections) guideline says instead of leaving them as they are?

Reply:
@ac-optimus
1. Yes, there is a list. When you open these docstyle pull requests, you should add the file to `setup.cfg`'s options for pydocstyle. The tests should only pass if the docstrings are using valid style (that's the point of these changes). I will push a commit adding the files for this PR. Please replicate this for the other docstyle PRs that are open.
https://github.com/glotzerlab/signac/blob/beefdf823a56709d7fc5a845bcf1dbf72d033d10/setup.cfg#L17-L19
2. You don't need to add `.. deprecated::` to any methods or classes that have a `@deprecated` decorator, because the decorator will do that automatically for us! The decorator edits the `__doc__` attribute of the method/class to add that.

Reply:
@bdice thanks for the information. Will do that.